.github/workflows/console-test-run.yml

```diff
@@ -0,0 +1,59 @@
+name: console-test-run
+on:
+  workflow_call:
+    inputs:
+      github-sha:
+        required: true
+        type: string
+jobs:
+  console-test:
+    timeout-minutes: 5
+    runs-on: self-hosted-runner
+    steps:
+      - name: Checkout console test repo
+        uses: actions/checkout@v3
+        with:
+          repository: vegaprotocol/console-test
+          path: './console-test'
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.11'
+
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+
+      - name: Install dependencies
+        run: |
+          poetry install --no-root
+        working-directory: ./console-test
+
+      - name: load Binaries
+        run: |
+          poetry run python -m vega_sim.tools.load_binaries
+        working-directory: ./console-test
+
+      - name: pull console
+        run: |
+          poetry run docker pull ghcr.io/vegaprotocol/frontend/trading:${{ inputs.github-sha }}
```

Comment:
When do you push those images to the docker registry? I mean you need to make sure docker image is published before this pipeline is started.

Reply:
In the Ci-Cd workflow there is a publish dist:

```
 publish-dist:
    needs: lint-test-build
    name: '(CD) publish dist'
    # if: ${{ needs.lint-test-build.outputs.projects != '[]' }}
    uses: ./.github/workflows/publish-dist.yml
    secrets: inherit
    with:
      projects: ${{ needs.lint-test-build.outputs.projects }}
```

Would it be better to keep console-test seperate from the main ci-cd workflow for now to iron out any issues etc?

Reply:
I think it's up to you :) If you are fine, the pipeline is in single place it's fine :) You can refactor it later if you think there is something not right :)